lib/chef_zero/endpoints/policy_revision_endpoint.rb

```diff
@@ -7,7 +7,25 @@ class PolicyRevisionEndpoint < RestBase
       # GET /organizations/ORG/policies/NAME/revisions/REVISION
       def get(request)
         data = parse_json(get_data(request))
-        data = ChefData::DataNormalizer.normalize_policy(data, request.rest_path[3], request.rest_path[5])
+        data[:policy_group_list] = Array.new
+
+        request_policy_name = request.rest_path[3]
+        request_policy_revision = request.rest_path[5]
+        request.rest_path[2] = "policy_groups"
+        request.rest_path = request.rest_path.slice(0,3)
+
+        list_data(request).each do |group_name|
+          group_path = request.rest_path + [group_name]
+          policy_list = list_data(request, group_path + ["policies"])
+          policy_list.each do |policy_name|
+            revision_id = parse_json(get_data(request, group_path + ["policies", policy_name]))
+            if (policy_name == request_policy_name) && (revision_id == request_policy_revision)
+              policy_group_list = data[:policy_group_list]
+              data[:policy_group_list] = policy_group_list + [group_name]
```

Comment:
could we add some description about this? eg: what is the changes in response after groups are included. How does it look with no groups. etc.